babel/src/test/resources/sql/big-query.iq

```diff
@@ -398,14 +398,12 @@ SELECT
 # In the following example, EXTRACT returns values corresponding to
 # different date parts from a column of dates near the end of the
 # year.
-
-!if (false) {
 SELECT
   d,
   EXTRACT(ISOYEAR FROM d) AS isoyear,
-  EXTRACT(ISOWEEK FROM d) AS isoweek,
-  EXTRACT(YEAR FROM d) AS year,
-  EXTRACT(WEEK FROM d) AS week
+  EXTRACT(ISOWEEK FROM d) as isoweek,
+  EXTRACT(WEEK(TUESDAY) FROM d) AS week_tues,
```

Comment:
can you add new columns instead of overriding existing column ?

Reply:
Yes I could. I did a poor job of noting this but the reason I didn't do this originally is because WEEK for the EXTRACT function is currently configured incorrectly in Avatica. As it stands, EXTRACT(WEEK ...) actually returns the ISOWEEK. This causes all tests for EXTRACT WEEK in Calcite to be off in some cases where the ISOWEEK and WEEK for a given date happen to be different. I can open a ticket for this issue if you think it is warranted?